ui/lib/apps/ContinuousProfiling/pages/Detail.tsx

```diff
@@ -58,53 +106,99 @@ export default function Page() {
         },
       },
       {
-        name: t('continuous_profiling.detail.table.columns.status'),
+        name: t('conprof.detail.table.columns.status'),
         key: 'status',
         minWidth: 150,
         maxWidth: 200,
         onRender: (record) => {
-          if (record.state === 'failed') {
-            return <Badge status="error" text={record.error} />
-          } else {
+          if (record.state === 'finished' || record.state === 'success') {
             return (
               <Badge
                 status="success"
-                text={t('continuous_profiling.detail.table.status.finished')}
+                text={t('conprof.detail.table.status.finished')}
               />
             )
           }
+          if (record.state === 'failed') {
+            return <Badge status="error" text={record.error} />
+          }
+          return <Badge text={t('conprof.list.table.status.unknown')} />
+        },
+      },
+      {
+        name: t('conprof.detail.table.columns.actions'),
+        key: 'actions',
+        minWidth: 150,
+        maxWidth: 200,
+        onRender: (record) => {
+          const rec = record as ConprofProfileDetail
+          let actionsKey = TEXT_ACTIONS
+          if (rec.profile_type !== 'goroutine') {
```

Comment:
I confirmed with @crazycs520 , it is different with manual profiling for mutex profile type. in manual profiling, it profiles with `debug=1` params and get txt result, it would affect the cluster performance, so in the conprof, we don't attach the `debug=1` params, then we get protobuf result.

Reply:
Got it! 👍